libs/odometry/multi_visual_odometry_base.cpp

```
@@ -73,12 +118,38 @@ bool MultiVisualOdometryBase::track(const Sources& curr_sources, [[maybe_unused]
   TRACE_EVENT ev = profiler_domain_.trace_event("MultiVisualOdometryBase::track()", profiler_color_);
   const int64_t timestamp = (*first_image)->get_image_meta().timestamp;  // current frame timestamp
   Isometry3T predicted_world_from_rig = prev_world_from_rig_;
+  Isometry3T world_from_rig;
```

Clang (14.0.6)

[warning] variable 'world_from_rig' is not initialized

(cppcoreguidelines-init-variables)